Source/AliveLibAE/Input.cpp

```diff
@@ -1379,37 +1392,48 @@ EXPORT int Input_Convert_KeyboardGamePadInput_To_Internal_Format_492150()
                             {
                                 pressed_keyboard_keys = keys_down;
                                 input_command_delete_pressed = 1;
+                                ++shoulderButtonsPressedCount;
                             }
                         }
+
+                        if (sGamePadBindings_5C98E0[i] & InputCommands::Enum::eRun)
+                            if ((1 << i) & pButtons)
+                                ++shoulderButtonsPressedCount;
+
+                        if (sGamePadBindings_5C98E0[i] & InputCommands::Enum::eSneak)
+                            if ((1 << i) & pButtons)
+                                ++shoulderButtonsPressedCount;
                     }
 
-                    if (input_command_c_pressed)
+                    // OG Change - chant with any shoulder button combo
+                    const bool isChanting = IsChantingAnyShoulderButton(shoulderButtonsPressedCount);
+                    // Original Method:
+                    IsChanting(input_command_c_pressed, input_command_delete_pressed);
```

Comment:
I'm confused because now it calls both?  prob need something in options.cmake for this?